index.js

```diff
@@ -93,15 +113,18 @@ module.exports.setLevel = function(level) {
 }
 
 /**
- * Change output type. Available values are "pretty" or "json". Default is json
- * @param {string} type
+ * Set outputs transport to use
+ * @param {Array<LogWriter>|LogWriter} outputs
  */
-module.exports.setOutput = function(type) {
-    if (!internals.outputs.includes(type)) {
-        throw new Error(`Invalid output: '${type}'`)
-    }
+module.exports.setOutput = module.exports.setOutputs = function(outputs) {
+    if (!outputs) outputs = []
+    if (!_.isArray(outputs)) outputs = [outputs]
+
+    _.forEach(outputs, output => {
```

Comment:
Same here, `forEach`, `typeof`…

Reply:
of for the forEach. I find isFunction much easier and since we are keeping lodash and i don't see the issue of using this function of lodash :)